active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,335 @@
+      Margin="0, 10, 0, 0"
+      NumberOfPages="{x:Bind Pictures.Count}"
+      SelectedPageIndex="{x:Bind Path=Gallery.SelectedIndex, Mode=TwoWay}" />
+</StackPanel>
+```
+#### Pip and Button customization
+
+The navigational buttons and pips can be customized via the PreviousButtonStyle, NextButtonStyle, SelectedPipStyle, and DefaultPipStyle.
+As a note, setting the PreviousButtonStyle and NextButtonStyle can overwrite what is set in PreviousButtonVisibility and NextButtonVisibility.
+Additionally, the SelectedPipStyle and DefaultPipStyle can overwrite what is set in the theme resources.
+
+![A prototype of five dots, the PipsPager, laid out horizontally with caret-style navigational arrows](images/PipsPager_CustomNavButtons.png)
+
+XAML
+``` XAML
+<Page.Resources>
+    <Style x:Key="NavButtonBaseStyle" TargetType="Button" BasedOn="{StaticResource NavigatonButtonBaseStyle}">
+        <Setter Property="Width" Value="30" />
+        <Setter Property="Height" Value="30" />
+        <Setter Property="FontSize" Value="12" />
+    </Style>
+
+    <Style x:Key="PreviousButtonStyle" BasedOn="{StaticResource NavButtonBaseStyle}" TargetType="Button">
+        <Setter Property="Content" Value="&#xEDDB;" />
+    </Style>
+
+    <Style x:Key="NextButtonStyle" BasedOn="{StaticResource NavButtonBaseStyle}" TargetType="Button">
+        <Setter Property="Content" Value="&#xEDDC;" />
+    </Style>
+</Page.Resources>
+
+<muxc:PipsPager x:Name="CustomNavButtonPipsPager"
+  PreviousButtonStyle="{StaticResource PreviousButtonStyle}"
+  NextButtonStyle="{StaticResource NextButtonStyle}" />
+```
+## PipsPager member notes
+
+| Name | Description| Default | 
+|:---:|:---|:---|
+| NumberOfPages | Sets the max number of pages the index control will iterate through. The default will represent an infinite page range. | -1
+| Previous and Next Style | Give the developer the option to customize the style by changing the text or glyph for the edge buttons. | N/A
+| SelectedPageIndex | The 0 based index that is currently selected. It will default to the first index. | 0
+| MaxVisiblePips | The maximum number of pips that will appear in the control at once. If the NumberOfPages is greater than this value, the UI will scroll so that the selected pip is centered. | 5
+| Orientation | The orientation of the control. Can be either Vertical or Horizontal. | Horizontal
+| PipsPagerSelectedIndexChangedEvent | Event that is fired after the user selects a pip or the directional buttons. This event will return the index number of the page that the user selected. | N/A
+
+## PipsPager Theme Resources
+
+You can modify the look of a PipsPager by specifying Xaml resources in your app.
+For more info, see the
+[lightweight styling guide](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling).
+
+| Resource key | Description | Type
+|:-:|:--| :-- |
+| PipsPagerButtonWidth | Sets the bounding box width for each pip | Double |
```

Comment:
Wanted to highlight this issue [#4071](https://github.com/microsoft/microsoft-ui-xaml/issues/4071) that likely impacts the button width and height theme resources. Having separate theme resources for both vertical and horizontal layout orientation is a pattern we introduced previously with InfoBar. 

Would you recommend we follow that pattern here or keep the current theme resources and make changes in the code behind to support orientation switching?

Reply:
If design calls for different values across horizontal and vertical, we should expose them as separate theme resources.

Reply:
Recommend: Add theme resources. FooHeightWhenVertical, FooHeightWhenHorizontal? (Follow InfoBar precedence).

Reply:
Updated spec to reflect the changes proposed in issue above. @beervoley can you link your PR here when opened? Thanks!